sfaira/data/dataloaders/loaders/d10_1038_s41586_019_1289_x/homosapiens_brain_2019_10x3v2_velasco_001_cell_type.tsv

```diff
@@ -0,0 +1,17 @@
+source	target	target_id
+Astroglia	astrocyte	CL:0000127
+CFuPNs	pyramidal neuron	CL:0000598
+CPNs	pyramidal neuron	CL:0000598
+Cajal-Retzius	Cajal-Retzius cell	CL:0000695
+Cycling	neural progenitor cell	CL:0011020
+IPCs	progenitor cell	CL:0011026
+IPCs/Immature PNs	progenitor cell	CL:0011026
+Immature CFuPNs	pyramidal neuron	CL:0000598
+Immature CPNs	pyramidal neuron	CL:0000598
+Immature Interneurons	interneuron	CL:0000099
+Immature PNs	pyramidal neuron	CL:0000598
+RG	radial glial cell	CL:0000681
+Unknown	unknown	unknown
+Ventral Precursors	precursor cell	CL:0011115
```

Comment:
I'd map ventral precursor to "neural progenitor cell"

Reply:
done